Gemfile

```diff
@@ -10,7 +10,6 @@ gem "govuk_publishing_components"
 gem "govuk_sidekiq"
```

Comment:
Thanks @thomasleese - I realise I should have explicitly mentioned this one can go too.

Reply:
Good point... that's gone too now!